historical-data/rename.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+# remove the symlinks first, we'll recreate them later
+rm common/*
+
+# find and rename all the historical files
+find . -name *testgrid* -exec rename  -- "-grid=old&show-stale" "-show-stale" {} \;
```

Comment:
I don't speak bash, but I'm guessing this is some kind of a replacement thing

Reply:
"find all files in the current directory tree that contain the string `testgrid`, for each file(`{}`) execute the `rename` command, replacing $1 with $2 in the filename".  The `--` is so that `-grid` doesn't get treated as an argument to rename.